interface.py

```diff
@@ -84,14 +85,14 @@ def index():
                 files = music_library.get_files_recursively(folder)
             else:
                 files = music_library.get_files(folder)
-            files = list(map(lambda file: ('file', folder + '/' + file), files))
+            files = list(map(lambda file: ('file', folder + '/' + file, datetime.now()), files))
```

Comment:
instead of `datetime.now()` i would use `datetime.now().timestamp()` so that the output to the user is cleaner (the same as in `line 72`)